defold-rive/src/comp_rive.cpp

```diff
@@ -114,6 +113,7 @@ namespace dmRive
     struct RiveWorld
     {
         CompRiveContext*                    m_Ctx; // JG: Is this a bad idea?
+        rive::HRenderer                     m_RiveRenderer;
```

Comment:
The question is what should be kept in CompRiveContext? Should something else be moved to RiveWorld as well?